### DISCUSSION TITLE
Three options all labeled "volume mixer" is too confusing. Please make them distinct.

### DISCUSSION THREAD

**Discussion by stamminator** [File-New-Project/EarTrumpet#1571] [Completed ideas]
Please consider making the various options in the EarTrumpet menu each labeled "volume mixer" distinct. I always end up wasting time clicking and closing one or two options before I find the one I want.

![i43vr4qsAi](https://github.com/File-New-Project/EarTrumpet/assets/10633170/8b37c586-c15e-4d69-b8bb-0eeaaaa2374c)

My suggestion would be to make the legacy Windows one say "Legacy Volume Mixer", the Windows 11 one say either just "Volume Mixer" or "System Volume Mixer", and the EarTrumpet one say "EarTrumpet Volume Mixer"

**Comment by riverar** [comment]
@stamminator Thanks, I created an issue to track this (https://github.com/File-New-Project/EarTrumpet/issues/1572). This is an unfortunate side effect of the OS using different/similar names for these areas.
